single-research_project.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+use UCFResearchPublication\Common;
+
+$principle_investigator    = get_field( 'rp_principle_investigator', $post->ID );
+$investigators = get_field( 'rp_coprinciple_investigators', $post->ID );
+
+get_header(); the_post(); ?>
+
+<article class="<?php echo $post->post_status; ?> post-list-item">
+
+    <div class="container mt-4 mt-sm-5 mb-5 pb-sm-4">
+		<div class="row">
+			<div class="col-lg-8">
+				<?php the_content(); ?>
+            </div>
+
+            <?php if ( $principle_investigator || $investigators ) : ?>
+			<div class="col-lg-4">
```

Comment:
Consider adding `mt-4 mt-lg-0` to this column for additional breathing room at -md and lower